### ISSUE TITLE
Bootstrap-email styles applied to main application

### ISSUE THREAD

**Issue by JenniferMB** [stuyam/bootstrap-email#33]
I have installed the Bootstrap-email gem according to instructions and my main application styling has changed due to the addition of the styles. I assumed the styles of this gem would only be applied to the mailer where I specified the stylesheet link, but that is not the case.
Unfortunately I cannot provide an example due to workplace constraints, but I am hoping some insight might be available with the meager amount of information I am providing. Thank you.

**Comment by stuyam** [comment]
Look at this page: https://bootstrapemail.com/docs/setup
You should be creating a new application stylesheet called `application-mailer.scss` for example and adding `@import 'bootstrap-email';` to import the bootstrap email css. You should not be adding it to the main `application.css` file.

Is that not working for you?

**Comment by JenniferMB** [comment]
I chose a different filename (email.css.scss), created the file in my
app/assets/stylesheets folder, and added the @import 'bootstrap-email'
directive. I configured my config/initializers/assets.rb to append the file
along with the other precompile assets, added my <&= "stylesheet_link_tag
'email', media: 'all' " %> statement in the head section of my
mailer.html.erb file, ran rake asset:clobber and rake asset:precompile, and
successfully tested my mailer.
What I can't figure out is how the bootstrap mailer styles are getting
wrapped up with my application.css.scss.
It is true that both style files are in the same folder
app/assets/stylesheets, is that the reason? I do think I have followed the
process as directed in the documentation, am I missing something?

**Comment by JenniferMB** [comment]
I think I solved it.
I have been studying the Ruby on Rails guide for The Asset Pipeline, and
came across a few hints. I renamed my file from email.css.scss to
email.scss and moved it from app/assets/stylesheets to
lib/assets/stylesheets. Now my application is not including the email
styles, and so far my mailer is still working properly.
So I think in this case the answer is to use the lib/assets directory
instead of the app/assets directory for the bootstrap-email styles, because
Sprockets is going to precompile the subdirectories of app/assets whether
you like it or not.
I am very sorry to have troubled you, I will close my issue as solved.
